spec/rails_helper.rb

```diff
@@ -5,6 +5,7 @@
 require 'rspec/rails'
 require 'shoulda/matchers'
 require 'cancan/matchers'
+include DeviseMasquerade::Controllers::UrlHelpers
```

Comment:
The url for masquerades are not defined in specs, this is for including them, you want me to just include that in those specs who need it ?